ts/indicators/ao.src.ts

```diff
@@ -0,0 +1,260 @@
+/* *
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+'use strict';
+
+import H from '../parts/Globals.js';
+
+/**
+ * Internal types
+ * @private
+ */
+declare global {
+    namespace Highcharts {
+        class AoIndicator extends SmaIndicator {
+            public nameBase: string;
+            public data: Array<AoIndicatorPoint>;
+            public nameComponents: Array<string>;
+            public options: AoIndicatorOptions;
+            public pointClass: typeof AoIndicatorPoint;
+            public points: Array<AoIndicatorPoint>;
+            public getValues(
+                series: Series,
+            ): (boolean|IndicatorValuesObject);
+            public drawGraph(): void;
+            public getColumnMetrics: ColumnSeries['getColumnMetrics'];
+            public crispCol: ColumnSeries['crispCol'];
+            public translate: ColumnSeries['translate'];
+            public drawPoints: ColumnSeries['drawPoints'];
+        }
+
+        interface AoIndicatorOptions extends SmaIndicatorOptions {
+            greaterBarColor?: ColorString;
+            lowerBarColor?: ColorString;
+            threshold?: number;
+            groupPadding?: number;
+            pointPadding?: number;
+            states?: SeriesStatesOptionsObject<AoIndicator>;
+        }
+
+        class AoIndicatorPoint extends SmaIndicatorPoint {
+            public series: AoIndicator;
+        }
+    }
```

Comment:
I also forgot it myself in an other module... one thing is missing. You have to add `ao` to the `SeriesTypesDictionary`.

Reply:
Fixed